firefox_ui_tests/remote/security/test_mixed_content.py

```diff
@@ -0,0 +1,49 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestMixedContent(FirefoxTestCase):
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.test_url = 'https://mozqa.com/data/firefox/security/mixedcontent.html'
+
+    def test_mixed_content(self):
+        with self.marionette.using_context('content'):
+            self.marionette.navigate(self.test_url)
+        favicon = self.browser.navbar.locationbar.favicon
+
+        favicon_image = self.marionette.execute_script("""
+                    return arguments[0].ownerDocument.defaultView
+                    .getComputedStyle(arguments[0])
+                    .getPropertyValue('list-style-image')
+            """, script_args=[favicon])
+
+        try:
+            is_mixed = favicon_image.index(
+                "identity-icons-https-mixed-display")
+            if is_mixed:
+                found = True
+        except Exception:
+            found = False
+        finally:
+            self.assertIs(found, True)
+
+        identity_popup = self.browser.navbar.locationbar.identity_popup
+        identity_box = identity_popup.box
+        identity_box.click()
+
+        def identity_popup_displayed(self):
+            return identity_popup.popup.is_displayed()
+
+        self.wait_for_condition(identity_popup_displayed)
+        encryption_label = identity_popup.encryption_label
+        label_text = encryption_label.text
+        prop = self.browser._l10n.get_property(
+            ['chrome://browser/locale/browser.properties'],
+            'identity.broken_loaded'
+            )
```

Comment:
@chmanchester No it does not. I tried that before using ._l10n.get_property

| MarionetteException: MarionetteException: DTD Entity not found: identity.broken_loaded